### PR TITLE
[OSCI]add new author tiffanylamm

### PR DESCRIPTION
**Pull request by tiffanylamm**
### Description
add tiffanylamm to _author
### Issues Resolved
none

### Check List
- [x ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/tiffanylamm.markdown [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Tiffany Lam 
+short_name: tifflam
+github: tiffanylamm
+---
+
+**<Tiffany Lam>** <I am a junior at UC Berkeley!>
```
Comment (ashwin-pc) [inline _authors/tiffanylamm.markdown:7]:
```suggestion
**Tiffany Lam** I am a junior at UC Berkeley!
```
Can you make this change?

**Review by ashwin-pc — CHANGES_REQUESTED** [review]
@tiffanylamm  can you move this file to `_authors/osci_2023`? Sorry there was an error in my notes.

File: _authors/tiffanylamm.markdown (hunk 1 above)
Comment (tiffanylamm) [reply _authors/tiffanylamm.markdown:7]: got it. i'll move it over.

Comment (tiffanylamm) [reply _authors/tiffanylamm.markdown:7]: @ashwin-pc I've made the above changes. Do I need to make new PR?

Comment (joshuarrrr) [reply _authors/tiffanylamm.markdown:7]: @tiffanylamm If you make your changes on the same branch and then `git push`, it should update this PR automatically.

Comment (dtaivpp) [reply _authors/tiffanylamm.markdown:7]: @tiffanylamm hey, wanted to checkin and remind you to push your changes so we can get this merged!

Comment (tiffanylamm) [reply _authors/tiffanylamm.markdown]: I've pushed my changes. Sorry for the hold up I thought pushed it correctly but I didn't.

**Review by ashwin-pc — APPROVED** [review]
LGTM :)

**Review by joshuarrrr — APPROVED** [review]
LGTM

**Pull request merged**